astroquery/mast/tests/test_mast_remote.py

```diff
@@ -0,0 +1,50 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+
+from ... import mast
+
+@remote_data
+class TestMast:
```

Comment:
Can you clarify?  I am not getting any errors running the tests in either 2 or 3.